treeherder/client/thclient/client.py

```diff
@@ -749,6 +753,21 @@ def get_repositories(self):
         """
         return self._get_json(self.REPOSITORY_ENDPOINT, None)
 
+    def get_products(self):
+        """
+        Get a list of valid treeherder products.
```

Comment:
Let's remove "valid" here. Some of the products we're storing are probably invalid. ;)